sql/init_db.sql

```diff
@@ -11,8 +11,51 @@ create table if not exists users
     "updatedAt" timestamp with time zone not null
 );
 
+create table if not exists groups
```

Comment:
Would be nice to use sequelize-cli and create migrations

Reply:
https://www.youtube.com/watch?v=56O93Il5sTs&ab_channel=%D0%9A%D0%B0%D0%BD%D0%B0%D0%BBCarefool%27%D0%B0